src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.db.FruitShopDaoImpl;
+import core.basesyntax.impl.AmountFromFileImpl;
+import core.basesyntax.impl.ConvertingFileToFruitTransactionImpl;
+import core.basesyntax.impl.FruitsFromFileImpl;
+import core.basesyntax.impl.ReadingCsvFileImpl;
+import core.basesyntax.impl.WritingToNewFileImpl;
+import core.basesyntax.service.AmountFromFile;
+import core.basesyntax.service.ConvertingFileToFruitTransaction;
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.FruitsFromFile;
+import core.basesyntax.service.ReadingCsvFile;
+import core.basesyntax.service.WritingToNewFile;
+import java.util.List;
+
+public class Main {
+    // HINT: In the `public static void main(String[] args)`
+    // it is better to create instances of your classes,
+    // and call their methods,
+    // but do not write any business logic in the `main` method!
+    public static void main(String[] args) {
+        FruitShopDao fruitShopDao = new FruitShopDaoImpl();
```

Comment:
The instantiation of the services should not be done directly in the `main()` method. Consider using a dependency injection mechanism or factory pattern to create instances of these services.